webpack.common.config.js

```diff
@@ -536,9 +690,10 @@ module.exports = Merge.smart({
             poll: true
         },
 
-        node: {
-            fs: 'empty'
-        }
+        // https://stackoverflow.com/questions/64361940/webpack-error-configuration-node-has-an-unknown-property-fs
+        // node: {
+        //     fs: 'empty'
+        // }
```

Comment:
Did you try the suggested fix in there?

Reply:
It wasn't clear to me how this was being used. I didn't see a difference when having the suggestion in resolve or not. I'll go ahead and add it there for now.

Reply:
went on a bit of a wild ride with this one

https://github.com/openedx/edx-platform/blob/0e5f6897ed8dc0a9199aa55bebe1e8344681e502/webpack.common.config.js#L671-L683